packages/data-context/src/sources/WizardDataSource.ts

```diff
@@ -36,13 +40,32 @@ export class WizardDataSource {
         name: storybookDep,
         description: PACKAGES_DESCRIPTIONS[storybookDep],
         package: storybookDep,
+        installer: '',
       })
     }
 
     return packages
   }
 
-  async resolvePackagesToInstall (): Promise<string[]> {
+  async installDependenciesCommand () {
+    const commands = {
+      'npm': 'npm install -D ',
+      'pnpm': 'pnpm install -D ',
+      'yarn': 'yarn add -D ',
+    }
+
+    const deps = (await this.ctx.wizard.packagesToInstall() ?? [])
+    .map((pack) => `${pack.installer}`)
+    .join(' ')
+
+    return `${commands[this.ctx.coreData.packageManager ?? 'npm']} ${deps}`
+  }
+
+  async installedPackages (): Promise<string[]> {
```

Comment:
This is so we don't actually need to run a `npm install` during E2E tests; we simulate that by setting this to pretend like we ran `npm install`.